datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -10078,7 +10078,7 @@ type MLModelProperties {
     """
     The display name of the model used in the UI
     """
-    name: String!
+    name: String
```

Comment:
this was added in the last PR but i noticed that this causes error when fetching mlmodel via graphQL therefore causing UI break : 

was declared as a non null type, but the code involved in retrieving data has wrongly returned a null value.  The graphql specification requires that the parent field be set to null, or if that is non nullable that it bubble up null to its parent and so on.


let me know if this is the right approach to fix this @jjoyce0510 @RyanHolstien